src/App.scss

```diff
@@ -1,8 +1,7 @@
-:root {
-    --darkgrey: #444444;
-    --grey: #c2c2c2;
-    --lightblack: #232323;
-}
+$darkgrey: #444444;
+$grey: #c2c2c2;
+$lightblack: #232323;
+$grey-border: 1px solid grey;
```

Comment:
You should also consider adding text sizes for variables in addition to the colors. Something like the example below might help when keeping sizes consistent across the site.

```
    $text-sm: 1.25rem;
    $text-md: 1.75rem;
    $text-lg: 2.5rem;
    ...
````